sdk/storage/Azure.Storage.Blobs.Batch/src/BlobBatchClient.cs

```diff
@@ -102,16 +102,17 @@ public BlobBatchClient(BlobServiceClient client)
         public BlobBatchClient(BlobContainerClient client)
         {
             Uri = client.Uri;
-            Pipeline = BlobContainerClientInternals.GetHttpPipeline(client);
-            BlobClientOptions options = BlobContainerClientInternals.GetClientOptions(client);
+            var blobServiceClient = client.GetParentBlobServiceClient();
```

Comment:
nit - use `BlobServiceClient` instead of `var`

Reply:
ok. would look better given surrounding. However, I'm not a big fan of using explicit types in context where type is obvious.